mappings/net/minecraft/predicate/entity/EntityPredicate.mapping

```diff
@@ -13,16 +13,38 @@ CLASS net/minecraft/unmapped/C_kvkvpjlm net/minecraft/predicate/entity/EntityPre
 	FIELD f_ycxvlbvr equipment Ljava/util/Optional;
 	FIELD f_yvrkjogt vehicle Ljava/util/Optional;
 	METHOD m_cviarbxt toJson ()Lcom/google/gson/JsonElement;
+	METHOD m_lenpszae fromBuilder (Lnet/minecraft/unmapped/C_kvkvpjlm$C_fnhnavbg;)Lnet/minecraft/unmapped/C_ctsfmifk;
+		ARG 0 builder
+	METHOD m_lvpiqxph fromJson (Lcom/google/gson/JsonElement;)Ljava/util/Optional;
```

Comment:
```suggestion
	METHOD m_lvpiqxph fromJsonOrEmtpy (Lcom/google/gson/JsonElement;)Ljava/util/Optional;
```

Reply:
```suggestion
	METHOD m_lvpiqxph fromJsonOrEmpty (Lcom/google/gson/JsonElement;)Ljava/util/Optional;
```
typo again :p

Reply:
woops, good catch(es)

Reply:
I don't like the `OrEmpty` addition because that's implied by the `Optional` return type. Also, these predicates are even stored in fields as `Optional`s, so it essentially is an integral part of the predicate.

Reply:
fair